src/DssSpell.t.base.sol

```diff
@@ -865,6 +861,13 @@ function checkIlkClipper(
         assertEq(clipper.wards(address(clipMom)), 1);
         assertEq(clipper.wards(address(esm)), 1);
 
+        try pip.bud(address(clipMom)) returns (uint256 bud) {
```

Comment:
This is actually checked in `checkIlkIntegration`

Reply:
the change is referring to https://github.com/makerdao/spells-mainnet/blob/master/archive/2021-06-04-DssSpell/DssSpell.t.sol#L2351

Reply:
We defined to remove these checks for now.

Reply:
I'm noticing that neither of these are getting called unless they are specifically getting invoked by the dev in the weekly test.  I'm ok with putting them in both places because it's not clear which one will be invoked in any given week.

Reply:
Well I agree on that, but adding just those two is too random, why not the end and spotter whitelist are checked?

Reply:
done in fa2d065